### PR TITLE
Refactor: remove stripe payment element subscription module

### PR DESCRIPTION
**Pull request by jonwaldstein**
## Description

This removes the StripePaymentElementSubscriptionModule from the feature plugin in favor of loading from give-recurring.

## Affects

- StripePaymentElementSubscriptionModule now lives in give-recurring

## Visuals

N/A

## Testing Instructions

- Since this is dropping functionality from here, will need to test this from the side of give-recurring and make sure there are no conflicts.

## Pre-review Checklist

-   [ ] Acceptance criteria satisfied and marked in related issue
-   [ ] Relevant `@unreleased` tags included in DocBlocks
-   [ ] Includes unit tests
-   [ ] Reviewed by the designer (if follows a design)
-   [ ] [Self Review](https://give.gitbook.io/development-manual/devops/github/code-reviews#self-review) of code and UX completed

### CODE REVIEW

File: src/PaymentGateways/Gateways/ServiceProvider.php [hunk 1]
```diff
@@ -109,13 +109,6 @@ private function registerGateways()
          * This module will eventually live in give-recurring
          */
         if (defined('GIVE_RECURRING_VERSION') && GIVE_RECURRING_VERSION) {
-            add_filter(
-                sprintf("givewp_gateway_%s_subscription_module", StripePaymentElementGateway::id()),
-                static function () {
-                    return StripePaymentElementGatewaySubscriptionModule::class;
```
Comment (glaubersilva) [inline src/PaymentGateways/Gateways/ServiceProvider.php:115]: As we don't need the `StripePaymentElementGatewaySubscriptionModule` class here, we also can remove it from the imports at the top of this file. I noticed that the `TestOffsiteGateway` is also present in the imports but isn't used in the file, so it also can be removed.

**Review by glaubersilva — CHANGES_REQUESTED** [review]
@jonwaldstein I left a comment related to a minor change. Taking that away looks good!

File: src/PaymentGateways/Gateways/ServiceProvider.php (hunk 1 above)
Comment (jonwaldstein) [reply src/PaymentGateways/Gateways/ServiceProvider.php:115]: Resolved 84e4f780

**Review by glaubersilva — APPROVED** [review]
@jonwaldstein Thanks! Looks good to QA.

**Pull request merged**